website/src/pages/index.js

```diff
@@ -259,20 +259,23 @@ const NewsletterSection = () => {
 };
 
 const Contributor200Poster = () => {
-  const [display, setDisplay] = useState(true);
+  const [display, setDisplay] = useState(false);
   const [theme, setTheme] = useState('light');
 
   useEffect(() => {
     addEventListener('click', () => {
-      if (localStorage.getItem('theme') !== theme) {
-        setTheme(localStorage.getItem('theme'));
-      };
+      setTheme(localStorage.getItem('theme'));
     });
-  }, [theme]);
-
-  if (!display || (typeof window !== 'undefined' && localStorage.getItem('SHOW_200_CONTRIBUTOR_EVENT_ENTRY'))) {
-    return null;
-  }
+    if (localStorage.getItem("theme")) {
+      setTheme(localStorage.getItem('theme'));
+    };
+    if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
Hi, is there more to the description here? I'm not quite sure why we need to listen to themes in this way

Reply:
1. Switch the operating system to dark mode.
2. open `apisix.apache.org`.
3. website is in dark mode but this component is light.